autodocs.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+LINT=
+GENERATE=
+
+while getopts :lL:gG: opt
+do
+    case "${opt}" in
+        l)
+            LINT=1
+            ;;
+        g)
+            GENERATE=1
+            ;;
+        \?)
+            echo "ERROR: unknown parameter \"$OPTARG\""
+            exit 1
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+IFS=$'\n'
+# create an array of all unique directories containing .tf files 
+arr=($(find . -name '*.tf' | xargs -I % sh -c 'dirname %' | sort -u))
+unset IFS
+
+for i in "${arr[@]}"
+do
+    docs_dir=$i/_docs
```

Comment:
directories may have spaces, need to quote the vars, use string interpolation...
```suggestion
    docs_dir="${dir}/_docs"
```

Reply:
fixed